src/storage/rocksdb/src/main/java/org/locationtech/geogig/rocksdb/RocksConnectionManager.java

```diff
@@ -178,6 +181,13 @@ private ColumnFamilyDescriptor newColDescriptor(String name) {
 
     private ColumnFamilyOptions newColFamilyOptions() {
         ColumnFamilyOptions colFamilyOptions = new ColumnFamilyOptions();
+
+        // enable bloom filter to speed up RocksDB.get() calls
+        BlockBasedTableConfig tableFormatConfig = new BlockBasedTableConfig();
```

Comment:
This probably should have come in as a separate PR -- but its fine.

Reply:
yeah